core/invoker/src/main/scala/whisk/core/container/ContainerUtils.scala

```diff
@@ -92,14 +92,13 @@ trait ContainerUtils {
         val cpuArg = Array("-c", cpuShare.toString)
         val memoryArg = Array("-m", s"${limits.memory.megabytes}m", "--memory-swap", s"${limits.memory.megabytes}m")
         val capabilityArg = Array("--cap-drop", "NET_RAW", "--cap-drop", "NET_ADMIN")
-        val consulServiceIgnore = Array("-e", "SERVICE_IGNORE=true")
```

Comment:
Needs to be changed in the new pool as well, see `DockerContainer.scala` line 75 (you probably didn't get it as a match in your directory-wide "consul" search? 😆 )